cmd/minishift/cmd/openshift/registry.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openshift
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/minishift/minishift/pkg/minishift/openshift"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/cobra"
+)
+
+var registryCmd = &cobra.Command{
+	Use:   "registry [flags]",
+	Short: "Prints the docker registry endpoint information.",
+	Long:  `Prints the openshift docker registry service endpoint information.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		registryInfo, err := openshift.GetDockerRegistryInfo()
+		if err != nil {
+			atexit.ExitWithMessage(1, err.Error())
+		} else {
```

Comment:
No need of `else` block here because if `err` is not nil then process get terminated.